packages/java-parser/scripts/single-sample-runner.js

```diff
@@ -1,9 +1,7 @@
 /**
  * This Script is used to debug the parsing of **small** code snippets.
  */
-"use strict";
-
-const javaParserChev = require("../src/index");
+import javaParserChev from "../src/index";
 
 const input = `
```

Comment:
If I run this script with the following code snippet, I have the following error, do you have the same ?

```js
import javaParserChev from "../src/index.js";

const input = `
  public class T {
  
          void t() {
              var a = "toto";
          }
  
  }
  `;

javaParserChev.parse(input, "compilationUnit");
```

Error:
> Ambiguous Alternatives Detected: <0, 1> in <OR> inside <localVariableType> Rule,
<'var', Identifier, '='> may appears as a prefix path in all these alternatives.
See: https://chevrotain.io/docs/guide/resolving_grammar_errors.html#AMBIGUOUS_ALTERNATIVES
For Further details.

Reply:
It does seem to appear in the tests also: https://github.com/jhipster/prettier-java/actions/runs/6835796470/job/18589989957?pr=616#step:6:13722

Reply:
Should we ignore the logging message in the LLStarLookaheadStrategy ?

```js
new LLStarLookaheadStrategy({ logging: () => {}}),
```

Reply:
Yeah I noticed these ambiguity logs in the test output, forgot to ensure they wouldn't be logged during production (I thought this would be covered by `skipValidations`, but I guess that only covers initialization validations). It would be nice if we could disable these logs for production, but leave them enabled for our tests (since it would be good to resolve them in the future). Perhaps with something like this?
```js
new LLStarLookaheadStrategy({
  logging: getSkipValidations() ? () => {} : undefined
}),
```